pkg/Cpanel/Security/Advisor/Assessors/Mysql.pm

```diff
@@ -79,14 +78,17 @@ sub _check_for_db_test {
     if ( !$exists ) {
         $self->add_good_advice(
             'key'  => 'Mysql_test_database_does_not_exist',
-            'text' => $self->_lh->maketext("MySQL test database doesn't exist.")
+            'text' => $self->_lh->maketext("[asis,MySQL] test database doesn’t exist.")
```

Comment:
Minor:  Vim will not be happy about this word.

In general, I was under the impression that contractions were typically frowned upon in formal writing - let alone in translated writings.  It might be worthwhile to address this technical debt now

Reply:
Sure, I can do that.